sdk/eventhubs/microsoft-azure-eventhubs-eph/pom.xml

```diff
@@ -14,7 +14,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.microsoft.azure</groupId>
   <artifactId>azure-eventhubs-eph</artifactId>
-  <version>3.2.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-eph;current} -->
+  <version>3.2.0</version> <!-- {x-version-update;com.microsoft.azure:azure-eventhubs-eph;current} -->
```

Comment:
Jim has a script that updates all the version numbers so they are fixed in any other locations where it is used. If you open this file and update to 3.2.0 from -beta.1:
https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_data.txt#L27-L29
then run:
`python.exe .\eng\versioning\update_versions.py --update-type library --build-type data --skip-readme`

Reply:
Done!

Reply:
Thank you @conniey for pointing it out and @JamesBirdsall for using it.